Framework/API/src/Algorithm.cpp

```diff
@@ -32,6 +32,9 @@
 
 #include <map>
 
+// Index property handling template definitions
+#include "MantidAPI/Algorithm.tcc"
```

Comment:
There are explicit instantiations in ArrayProperty.cpp but for some reason I keep getting the warnings. The code compiles, but with loads of warnings.